devel/olm/Portfile

```diff
@@ -18,13 +18,15 @@ homepage            https://gitlab.matrix.org/matrix-org/olm/
 
 master_sites        https://gitlab.matrix.org/matrix-org/olm/-/archive/${version}
 
-checksums           rmd160  04922bc2e9571698cbfb6c8f11be33e764e46dcf \
-                    sha256  1ca9926ce71d778fb7352d1ee77513194db8c7f49c0d69d38ac49ec3bafcea38 \
-                    size    521495
+checksums           rmd160  27b4c40d42bc81977fd432bf5be05c2bc09e23f8 \
+                    sha256  92ac1eccacbff620a1bc1a168ba204893d83bcb72646e456990ebe2480638696 \
+                    size    522692
 
 variant static description {Make static library} {
 	configure.args-append \
 		-DBUILD_SHARED_LIBS=NO
 }
 
+default_variants +static
```

Comment:
Why does this port have a static variant at all, and why do you want to make it the default? In MacPorts we want to offer dynamic libraries whenever we can. I recommend removing the variant, because if anyone uses it, it will break any software that depends on olm that was built when the variant was not selected.

Reply:
I think Nheko is the only dependant port right now, and it requires the static library. The olm build system builds either the static or dynamic library, not both, so I made the variant, and set static to default. If this isn't OK, I can change nheko to depend on olm-static, but I'm not sure how to specify dependence on a variant

Reply:
Looks like there was some issue on my laptop, causing cmake to look for the static version of libolm, instead of the dynamic one. I've removed this static variant.